src/wireviz/Harness.py

```diff
@@ -20,7 +20,7 @@
     Tweak,
     Side,
 )
-from wireviz.svgembed import embed_svg_images_file
+from wireviz.svgembed import embed_svg_images_file, embed_svg_images
```

Comment:
`isort` prefers this order:
```suggestion
from wireviz.svgembed import embed_svg_images, embed_svg_images_file
```
However, in this case, your PR will still be merged in soon, because we plan to do a final clean-up with `isort` and `black` before the release to fix reformatting issues like this.